extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -191,6 +236,7 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     if (this.layoutPromise_) {
       return this.layoutPromise_;
     }
+    this.lifecycleReporter_.sendPing('adRequestStart');
```

Comment:
OK, but probably with a different name. I imagine `adRequestStart` is when the real request get sent over the network.